docs/sparkr.md

```diff
@@ -663,13 +663,20 @@ Apache Arrow is an in-memory columnar data format that is used in Spark to effic
 
 ## Ensure Arrow Installed
 
-Currently, Arrow R library is not on CRAN yet [ARROW-3204](https://issues.apache.org/jira/browse/ARROW-3204). Therefore, it should be installed directly from Github. You can use `remotes::install_github` as below.
+Arrow R library is available on CRAN as of [ARROW-3204](https://issues.apache.org/jira/browse/ARROW-3204). It can be installed as below.
+
+```bash
+Rscript -e 'install.packages("arrow", repos="https://cloud.r-project.org/")'
+```
+
+If you need to install old versions, it should be installed directly from Github. You can use `remotes::install_github` as below.
```

Comment:
If installing old versions, above requireNamespace changes are still valid?

Reply:
Yea, it will work with the release. For old versions, only the CRAN check from dev won't work. I checked that CRAN passed in my local and `dapply` and `gapply` work manually.

Let me fix up tests if it still fails ..

Reply:
Yea, it will work because the workaround was used due to missing `arrow` CRAN. Since it's on CRAN, it will check.
I checked that CRAN passed in my local and `dapply` and `gapply` work manually.